grunt/custom/github-pre-release.js

```diff
@@ -40,13 +40,15 @@ module.exports = function( grunt ) {
 
 			const pluginVersion = grunt.file.readJSON( "package.json" ).yoast.pluginVersion;
 
-			// Open a text editor to get the changelog. For the first RC use the full changelog, for follow-up RCs only use the changes.
+			let initialContent = `Changes compared to ${grunt.config.get( "previousPluginVersion" )}:\n`;
```

Comment:
Not sure, but shouldn't you use straight quotes instead of backticks?

Reply:
These are fancy [JavaScript template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) 🙂.
They make string interpolation somewhat nicer.

If you look at the `logError` method above in this file, you can see it in action.

Reply:
Ah nice!